frontend.env

```diff
@@ -0,0 +1,6 @@
+VITE_FIREBASE_API_KEY = “AIzaSyCirQnmDKUSnkPTV454mell9uxAfCoyDe4”
```

Comment:
we should never expose Firebase or any sort of API key! Talk to @samshefnie about cleaning this